common/minimodel/__init__.py

```diff
@@ -437,7 +437,7 @@ class TooType(IntEnum):
     STANDARD = auto()
 
 
-class SetupTimeType(Enum):
+class SetupTimeType(IntEnum):
```

Comment:
My only concern with these is that I think we should use `Enum` if the ordering on them does not matter (i.e. we will not be sorting on these values or doing other ordering based comparisons) and `IntEnum` if the ordering is important: I was using this information to "tag" whether or not it was the case, but I think that this probably merits explicit documentation (on my part) in the actual class docstrings.